packages/atomic/src/components/ipx/atomic-ipx-embedded/atomic-ipx-embedded.tsx

```diff
@@ -37,6 +37,12 @@ export class AtomicIPXEmbedded implements InitializableComponent<AnyBindings> {
 
   @Event() animationEnded!: EventEmitter<never>;
 
+  @State() private hasFooterSlotElements = true;
+
+  public componentWillLoad(): void | Promise<void> {
+    this.hasFooterSlotElements = !!this.host.querySelector('[slot="footer"]');
```

Comment:
Could this be calculated from inside `atomic-ipx-body` instead ? I'm not sure it's possible, but it would be better since you would not need the `displayFooterSlot` attribute, it could all be handled internally in that component.